### DISCUSSION TITLE
Adding the Follow 3 Sites to the Server-Vote List

### DISCUSSION THREAD

**Discussion by InfinityofphoenixTimo** [gameserverapp/Platform#882] [Suggestions]
Add the following 3 pages to the supported server voting pages.

1. https://deutsche-arkserver.de/
2. https://xbox-arkservers.com/
3. https://ps4-arkserver.de/

They all have the same API endpoints.

**Comment by mvaessen** [comment]
same as #602, closing this one and will continue updating in the other item
